pkg/proc/core/core.go

```diff
@@ -145,14 +145,15 @@ type Process struct {
 	bi                proc.BinaryInfo
 	core              *Core
 	breakpoints       map[uint64]*proc.Breakpoint
-	currentThread     *LinuxPrStatus
+	currentThread     *Thread
 	selectedGoroutine *proc.G
 	allGCache         []*proc.G
 }
 
 type Thread struct {
-	th *LinuxPrStatus
-	p  *Process
+	th     *LinuxPrStatus
+	fpregs []proc.Register
```

Comment:
Because they are an attribute of a thread. LinuxPrStatus already contains the other CPU registers inside LinuxCoreRegisters, but both those structures are deserialized directly, so they can't be changed.